modules/logging-loki-gui-install.adoc

```diff
@@ -10,25 +10,25 @@ To install and configure logging on your {product-title} cluster, additional Ope
 
 .Prerequisites
 
-* Supported object store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
+* You have access to a supported object store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation).
+* You have administrator permissions.
 
 .Procedure
 
-. To install the {loki-op} in the {product-title} web console, click *Operators* -> *OperatorHub*.
+. In the {product-title} web console *Administrator* perspective, navigate to *Operators* -> *OperatorHub*.
 
-. Type {loki-op} in the filter by keyword box. Choose *Loki Operator* from the list of available Operators and click *Install*.
+. Type {loki-op} in the filter by keyword box. Click *Loki Operator* in the list of available Operators, then click *Install*.
 +
 [NOTE]
 ====
 The Community Loki Operator is not supported by Red Hat.
 ====
 
-. On the *Install Operator* page, for *Update channel* select *stable*.
+. Select *stable* or *stable-y.z* as the *Update channel*.
```

Comment:
Internally we use "y-stream" and "z-stream" to mean the Minor and Patch versions of the releases (I don't think I have heard "x-stream" yet...). Should the reference here reflect that?

```suggestion
. Select *stable* or *stable-x.y* as the *Update channel*.
```

Reply:
Ahhhh I see, I somehow missed the `5` 🤦🏻 apologies